frontend/src/Components/PostComponent.tsx

```diff
@@ -184,16 +185,18 @@ export default function PostComponent(props: PostComponentProps) {
 
                     <button onClick={toggleOptions} className={styles.options + ' ' + (showOptions ? styles.active : '')}><OptionsIcon /></button>
                     {showOptions &&
+                        <InterpreterWrapper content={props.post.content}>{(showAltTranslate, showAnnotate) =>
```

Comment:
Let's add `showAltTranslate` and `showAnnotate` into `useInterpreter` - we already have everything needed in there

Reply:
No, the whole purpose of this wrapper is to perform this logic on demand.  I'm actually concern about the overhead of `useInterpreter`, but I don't have any easy way to optimize it atm.